### PR TITLE
Add User model and get_user service method

### PR DESCRIPTION
**Pull request by zoidyzoidzoid**
Also some minor tidying.

The `User` model is almost identical to the `Ticket` model so I didn't add any tests.

### CODE REVIEW

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

File: teamsupport/models.py [hunk 1]
```diff
@@ -30,8 +32,13 @@ def actions(self):
             ) for action in actions.findall('Action')
         ], wrap=False)
 
+    @cached_property
+    def user(self):
+        user_data = self.client.get_user(self.UserID)
```
Comment (stefanor) [inline teamsupport/models.py]: User's `__init__` knows how to do that, so you could just pass the ID

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by stefanor** [comment]
:+1:

**Pull request merged**